change-validation/src/main/scala/com/normation/plugins/changevalidation/WorkflowService.scala

```diff
@@ -224,7 +225,12 @@ class TwoValidationStepsWorkflowServiceImpl(
         WorkflowAction("Validate", validatorActions)
 
       case Deployment.id     =>
-        WorkflowAction("Deploy", deployAction)
+        val deploymentAction = {
```

Comment:
https://github.com/Normation/rudder-plugins/commit/3103cf4338d98ed30425e4fbb5e31ba449433141#diff-98e8dc3343e38546c42491a39303758ad01214ed8e5ef40c6a2c4d85cfa73f8eL205 if think this is a oversight

Without this we cannot close the change-request, the change-request will be deployed, be will still remains in the deployment step